gke-deploy/deployer/deployer.go

```diff
@@ -150,9 +197,10 @@ func (d *Deployer) Prepare(ctx context.Context, images []name.Reference, appName
 		}
 	}
 
-	fmt.Printf("Saving hydrated resource configs to output: %q\n", output)
-	if err := resource.SaveAsConfigs(ctx, objs, output, d.Clients.OS); err != nil {
-		return fmt.Errorf("failed to save hydrated configs to output: %v", err)
+	hydratedOutput := filepath.Join(output, "hydrated")
```

Comment:
add a function in common to append the hydrated path and reuse it here and in apply